test/e2e/specs/all-features-confirmation/recordset/presentation.spec.js

```diff
@@ -271,6 +284,56 @@ describe('View recordset,', function() {
                 });
             });
 
+            it("should display the Export dropdown button with proper tooltip.", function() {
+                var exportDropdown = chaisePage.recordsetPage.getExportDropdown();
+                expect(exportDropdown.isDisplayed()).toBe(true, "The export dropdown button is not visible on the recordset app");
+                browser.actions().mouseMove(exportDropdown).perform();
+                var tooltip = chaisePage.getTooltipDiv();
+                chaisePage.waitForElement(tooltip).then(function () {
+                    expect(tooltip.getText()).toBe(testParams.tooltip.exportDropdown, "Incorrect tooltip on the export dropdown button");
+                    browser.actions().mouseMove(chaisePage.recordsetPage.getTotalCount()).perform();
+                });
+            });
+
+            it("should have '2' options in the dropdown menu.", function () {
+                chaisePage.recordsetPage.getExportDropdown().click().then(function () {
+                    expect(chaisePage.recordsetPage.getExportOptions().count()).toBe(2, "incorrect number of export options");
+                    return chaisePage.recordsetPage.getExportDropdown().click();
+                });
+            });
+
+            it("should have 'CSV' as a download option and download the file.", function() {
+                chaisePage.recordsetPage.getExportDropdown().click().then(function () {
+                    var csvOption = chaisePage.recordsetPage.getExportOption("CSV");
+                    expect(csvOption.getText()).toBe("CSV");
+                    return csvOption.click();
+                }).then(function () {
+                    return browser.wait(function() {
+                        return fs.existsSync(process.env.DOWNLOADS_PATH + '/Accommodations.csv');
+                    }, 10000);
+                }).then(null, function () {
+                    expect(false).toBeTruthy("Accommodations.csv was not downloaded");
+                });
+            });
+
+            it("should have 'BDBag' as a download option and download the file.", function() {
+                chaisePage.recordsetPage.getExportDropdown().click().then(function () {
+                    var bagOption = chaisePage.recordsetPage.getExportOption("BDBag");
+                    expect(bagOption.getText()).toBe("BDBag");
+                    return bagOption.click();
+                }).then(function () {
+                    return chaisePage.waitForElement(element(by.css(".export-progress")));
```

Comment:
I think we should move this `element(by.css(".export-progress")` to the chaisePage.